python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -60,17 +68,22 @@ class MTAirCompressorCsc(salobj.BaseCsc):
     def __init__(
         self,
         index: int,
+        config_dir: str = None,
         initial_state=salobj.State.DISABLED,
-        simulation_mode: int = 0,
+        override: str = "",
+        simulation_mode: valid_simulation_modes = 0,
     ):
         super().__init__(
             name="MTAirCompressor",
             index=index,
+            config_schema=CONFIG_SCHEMA,
+            config_dir=config_dir,
             initial_state=initial_state,
+            override=override,
             simulation_mode=simulation_mode,
         )
 
-        self.grace_period = 600  # TODO should be configurable - DM-35280
+        self.grace_period = self.host = self.port = self.unit = None
```

Comment:
Consider splitting this in individual assignments. It is really hard to read it.

Reply:
Splitted.